lib/lighthouse/veterans_health/client.rb

```diff
@@ -61,7 +61,7 @@ def list_observations
         get_list(first_response)
       end
 
-      def list_medications
+      def list_medication_requests
```

Comment:
I think you also have to update this clause on L45:

```unless %w[medications observations].include?(resource)```

Reply:
Good catch! Had to update a bunch of tests as well.